etc/php/php.ini

```diff
@@ -0,0 +1,4 @@
+date.timezone = "UTC"
```

Comment:
You are not using this configuration file because all configuration is in the image. Delete it.